docs/theme/head.hbs

```diff
@@ -0,0 +1,13 @@
+    <meta httpEquiv="Content-Language" content="en-US">
+    <meta name="apple-mobile-web-app-title" content="Starknet Foundry Book">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@swmansionxyz">
+    <meta name="twitter:creator" content="@jajakobyly">
+    <meta property="og:title" content="The Starknet Foundry Book">
+    <meta property="og:description" content="{{ chapter_title }}">
```

Comment:
Where is `chapter_title` defined?

Reply:
![Zrzut ekranu 2023-11-13 o 10 27 08](https://github.com/foundry-rs/starknet-foundry/assets/15279784/149f049a-5f03-4020-9830-3ae6d14b10aa)

These are examples of `chapter_title`. In the example, I added in the description section, there is how it looks for `Fork Testing`